images/hubble-proto/Makefile.defs

```diff
@@ -0,0 +1,4 @@
+# Copyright 2017-2020 Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+version := latest
```

Comment:
backporting this into 1.8, we'd have to adjust this version in the branch

Reply:
do we need this makefile if we are building images using dockerhub (and presumably tagging with branch name)? i'm a bit hesitant to add another file that needs to be updated as a part of the release process.

Reply:
This version string is still needed in the `api/v1/docker/Makefile`, so removing this makefile won't solve that particular issue. I'm ok dropping this makefile and let only dockerhub do builds.

Reply:
i see, i think you meant `api/v1/Makefile`. one option is to use https://github.com/cilium/cilium/blob/master/VERSION.

- should be straightforward for release branches. for example https://github.com/cilium/cilium/blob/v1.8/VERSION contains `1.8.2` so we can simply drop the patch version to get the tag name `1.8`.
- i guess the convention for the master branch is to set the patch version to `90` 🤯 so we need to check that and set the version to`latest`.

there is a similar logic in https://github.com/cilium/cilium/blob/master/Documentation/conf.py to infer branch name from the `VERSION` file.

Reply:
By this I assume the Makefile.defs? Because the image generation is only part of 1.9

Reply:
@aanm sorry for the late response, i mentioned `1.8` as an example. the question is whether we want to specify the version in `images/hubble-proto/Makefile.defs` or use the top-level `VERSION` file to avoid defining version in different places.

Reply:
I missed this discussion in the previous review.

So `hubble-proto` is a build-time only image, correct? In that case, I see it being very similar to e.g. the `builder` image and we should _not_ tie it to the Cilium version number. We should use a date as the tag.

Reply:
Yes, I think `builder` image is very similar. In fact, it already includes `protoc`, it just doesn't include `protoc-gen-go-*`, but I'd say it would be a good idea to add those there.

Reply:
> do we need this makefile if we are building images using dockerhub 

We are moving away from using hosted builds (see https://github.com/cilium/cilium/issues/11334).

Reply:
let's address this separately if needed. resolving for now.